grammar.ebnf

```diff
@@ -36,11 +38,40 @@ Statement = "{", {Statement}, "}"
 */
 LValue = Identifier, [ ".", Identifier ];
 
+Expression = ExpressionTerm 
+           | ParenthesisExpression
+           | FunctionCall
+           | NotExpression
+           | ArrayLookupExpression
+           | BinaryExpression
+
+BinOp = "+" | "-" | "*" | "/" | "%" | "&&" | "||" | "<" | "=<"| ">" | ">="| "=="| "!=";
+
+ParenthesisExpression = "(", Expression, ")";
```

Comment:
Better Parenthesized but you may want to write as Paren because it gets too verbose. Pick what you like.